edu.wpi.first.wpilib.plugins.core/src/main/java/edu/wpi/first/wpilib/plugins/core/WPILibCore.java

```diff
@@ -159,4 +160,20 @@ public static void logInfo(String msg) {
 	public static void logError(String msg, Exception e) {
 		getDefault().getLog().log(new Status(Status.ERROR, PLUGIN_ID, Status.OK, msg, e));
 	}
+	
+	void createUserDirs()
+	{
+		File directory = new File(WPILibCore.getDefault().getWPILibBaseDir() + "/user");
```

Comment:
We shouldn't be using hardcoded path separator strings. Use https://docs.oracle.com/javase/8/docs/api/java/nio/file/Paths.html#get-java.lang.String-java.lang.String...- to combine path elements.